test_deps/deps.ts

```diff
@@ -1,6 +1,6 @@
-import { ServerRequest } from "https://deno.land/std@0.92.0/http/server.ts";
+import { ServerRequest } from "https://deno.land/std@0.97.0/http/server.ts";
```

Comment:
why not leave it as `test_deps.ts`?

Reply:
Limitation of dmm tbh lol, it only updates deps.ts files and no other files with another file name

Reply:
sucks, but maybe we could add an issue in dmm to address something like this. or we can just put the test deps.ts file into the tests directory?

Reply:
> sucks, but maybe we could add an issue in dmm to address something like this. or we can just put the test deps.ts file into the tests directory?

ehhhh sure lol, i mean it makes sense to add this feature to dmm

also there isn't a 'root' tests dir, each middleware dir has its onw tests dir so not really possible

Reply:
LMFAO. woops. i totally overlooked that.